demo/image_demo.py

```diff
@@ -11,6 +11,7 @@ def parse_args():
     parser.add_argument('img', help='Image file')
     parser.add_argument('config', help='Config file')
     parser.add_argument('checkpoint', help='Checkpoint file')
+    parser.add_argument('--out-file', default='', help='Path to output file')
```

Comment:
Could we directly set the default as `None`?

Reply:
yes.